### PR TITLE
Ensuring that sync will return null when the pipeline does not exist

### PR DESCRIPTION
**Pull request by stjohnjohnson**
This seems like a strange problem to have.  I expected the datastore to throw an error, instead it just returns null.

### CODE REVIEW

File: test/lib/pipeline.test.js [hunk 1]
```diff
@@ -146,5 +148,14 @@ describe('Pipeline Model', () => {
                 done();
             });
         });
+
+        it('returns error if pipeline does not exist', (done) => {
```
Comment (d2lam) [inline test/lib/pipeline.test.js]: returns null?

**Comment by d2lam** [comment]
Sorry I think I wrote this code :(

File: test/lib/pipeline.test.js (hunk 1 above)
Comment (stjohnjohnson) [inline test/lib/pipeline.test.js]: Fixed

**Comment by tkyi** [comment]
👍 

giphy

**Comment by d2lam** [comment]
👍

**Pull request merged**